dataprofiler/tests/data_readers/test_csv_graph_data.py

```diff
@@ -184,14 +193,22 @@ def test_data_loader_edges(self):
             all_edges_present = True
             if not GraphData.is_match(input_file["path"], options):
                 return
-            data = GraphData(input_file["path"], options)
+            data = GraphData(input_file["path"], None, options)
```

Comment:
might notate this as data=None just so folks know when they are reading this what that param is

Reply:
`input_file_path=input_file["path"]), data=None, options`